<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetDateTimeForm</class>
 <widget class="QWidget" name="SetDateTimeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>60</y>
     <width>40</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>年：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxYear">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>65536</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>60</y>
     <width>40</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>月：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxMonth">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>50</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>65536</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>60</y>
     <width>40</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>日：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxDay">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>50</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>65536</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxSecond">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>50</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>65536</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxMinute">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>50</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>65536</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxHour">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>65536</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>120</y>
     <width>40</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>时：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>120</y>
     <width>40</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>分：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>120</y>
     <width>40</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>秒：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
